### ISSUE TITLE
[a11y] Separator headings should be as prominent as subview headings

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1237]
**Describe the bug**
Separators should appear as prominent as the subview headings:

**To Reproduce**
1. View a form
2. See subview heading
3. See separator

**Expected behavior**
How it appears in the current release:
![image](https://user-images.githubusercontent.com/37256050/163575807-5b8d67ab-3cb1-41a4-b851-c124a24a7754.png)

**Screenshots**
How it appears in a11y:
![image](https://user-images.githubusercontent.com/37256050/163575778-7cdd962d-029b-4520-b3db-cceb04e71c94.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Edge

**Comment by maxxxxxdlp** [comment]
Fixed in ce6a45e
